src/codemagic/utilities/argument_utils.py

```diff
@@ -0,0 +1,8 @@
+def get_binary_arguments_value(value, not_value):
```

Comment:
This function does not work in expected way if both arguments are `False`. It should raise, but instead returns `None`.

Reply:
I made it for our case when argument can only be `True` or `None`, but yes, will do in a more general way